donate/settings.py

```diff
@@ -19,15 +19,15 @@ class Config:
     LOG_FORMAT = "%(asctime)s | %(pathname)s:%(lineno)d |" \
         "%(funcName)s | %(levelname)s | %(message)s "
     SINGLE_CCY = True
+    CSRF_ENABLED = True
 
 
 class ProdConfig(Config):
     """ Production Configuration """
 
     ENV = 'prod'
     DEBUG = False
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL',
-                                             'postresql://localhost/example')
+    SQLALCHEMY_DATABASE_URI = os.environ.get('PROD_DATABASE_URI', None)
```

Comment:
Using `None` as the second argument to `.get` is redundant. `os.environ.get('PROD_DATABASE_URI', None)` is equivelant to `os.environ.get('PROD_DATABASE_URI')`.